src/cpp/DQM/serverext.cc

```diff
@@ -2172,7 +2188,11 @@ class VisDQMRenderLink
     assert(img.pngbytes.empty());
     img.busy = true;
     img.inuse++;
-    requestimg(img, jsonData, true);
+    if(jsroot){
+        requestimg(img, jsonData, false, true);
+    }else{
+        requestimg(img, jsonData, true);
+    }
```

Comment:
either it is jsroot or it's the old json format.
requestimg(img, jsonData, !jsroot, jsroot)
should be equivalent w/o branches